doc/sphinx/package_service_mapping.json

```diff
@@ -868,7 +896,121 @@
   "azure-mgmt-appconfiguration": {
     "category": "Management",
     "service_name": "App Configuration",
-    "manually_generated": true
+    "namespaces": [
+      "azure.mgmt.appconfiguration"
+    ]
+  },
+  "azure-mgmt-privatedns": {
+    "category": "Management",
+    "service_name": "Private DNS",
+    "namespaces": [
+      "azure.mgmt.privatedns"
+    ]
+  },
+  "azure-mgmt-frontdoor": {
+    "category": "Management",
+    "service_name": "Front Door",
+    "namespaces": [
+      "azure.mgmt.frontdoor"
+    ]
+  },
+  "azure-mgmt-hybridcompute": {
+    "category": "Management",
+    "service_name": "Hybrid Cloud",
+    "namespaces": [
+      "azure.mgmt.hybridcompute"
+    ]
+  },
+  "azure-mgmt-healthcareapis": {
+    "category": "Management",
+    "service_name": "Healthcare APIs",
+    "namespaces": [
+      "azure.mgmt.healthcareapis"
+    ]
+  },
+  "azure-mgmt-edgegateway": {
+    "category": "Management",
+    "service_name": "Edge Gateway",
+    "namespaces": [
+      "azure.mgmt.edgegateway"
+    ]
+  },
+  "azure-mgmt-deploymentmanager": {
+    "category": "Management",
+    "service_name": "Deployment Manager",
+    "namespaces": [
+      "azure.mgmt.deploymentmanager"
+    ]
+  },
+  "azure-mgmt-peering": {
+    "category": "Management",
+    "service_name": "Peering",
+    "namespaces": [
+      "azure.mgmt.peering"
+    ]
+  },
+  "azure-mgmt-appplatform": {
+    "category": "Management",
+    "service_name": "App Service",
+    "namespaces": [
+      "azure.mgmt.appplatform"
+    ]
+  },
+  "azure-mgmt-botservice": {
+    "category": "Management",
+    "service_name": "Bot Service",
+    "namespaces": [
+      "azure.mgmt.botservice"
+    ]
+  },
+  "azure-mgmt-costmanagement": {
+    "category": "Management",
+    "service_name": "Cost Management",
+    "namespaces": [
+      "azure.mgmt.costmanagement"
+    ]
+  },
+  "azure-mgmt-attestation": {
+    "category": "Management",
+    "service_name": "Attestation",
+    "namespaces": [
+      "azure.mgmt.attestation"
+    ]
+  },
+  "azure-mgmt-datashare": {
+    "category": "Management",
+    "service_name": "Data Share",
+    "namespaces": [
+      "azure.mgmt.datashare"
+    ]
+  },
+  "azure-mgmt-serialconsole": {
+    "category": "Management",
+    "service_name": "Serial Console",
+    "namespaces": [
+      "azure.mgmt.serialconsole"
+    ]
+  },
+  "azure-mgmt-storagesync": {
+    "category": "Management",
+    "service_name": "Storage",
```

Comment:
I called it `storage sync` - though i think there is something like storage sync services under `Azure File Sync` I am not sure what we should default it to. But let me know whatever we should keep, so that if need be i'll update in the JS repo too

Reply:
Given that Azure File Sync is required to work with Storage accounts, going to classify this under Storage for now.